addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -49,7 +49,13 @@
                     <div t-field="doc.client_order_ref">SO0000</div>
                 </div>
                 <div t-if="doc.date_order" class="col" name="informations_date">
-                    <strong t-if="doc.state in ['draft', 'sent']">Quotation Date</strong>
+                    <t t-set="is_proforma" t-value="env.context.get('proforma', False) or is_pro_forma"/>
+                    <strong
+                        t-if="doc.state in ['draft', 'sent'] and not is_proforma"
+                    >
+                        Quotation Date
+                    </strong>
+                    <strong t-elif="is_proforma">Issued Date</strong>
```

Comment:
Do conditions like it did for layout_document_title this way you don't have to check `not is_proforma` :thinking: